enterprise/backend/src/metabase_enterprise/metabot_v3/tools/api.clj

```diff
@@ -1154,22 +1155,31 @@
   [:and
    [:map
     [:transform_id :int]
-    [:source {:optional true} [:maybe ms/Map]]]
+    [:source {:optional true} [:maybe [:map
+                                       [:type {:optional true} :keyword]
```

Comment:
Are `:type` and `:query` really optional?

Reply:
Probably not. I was going off of the existing dependency check API schema: https://github.com/metabase/metabase/blob/22bce8c93e351276243d2206fd0724092e5ac8b4/enterprise/backend/src/metabase_enterprise/dependencies/api.clj#L78-L80. But apparently that was Cam's best guess. I'll look around and see if there's anything more authoritative